tests/Vagrantfile

```diff
@@ -37,32 +37,39 @@ Vagrant.configure("2") do |config|
 
   config.vm.synced_folder '../', '/chocoRoot'
 
-  config.vm.provision "shell", inline: <<-SHELL
+  config.vm.provision "shell", name: "prep", inline: <<-SHELL
     cscript c:/windows/system32/slmgr.vbs /rearm | Out-Null
     Set-ExecutionPolicy Bypass -Scope Process -Force
-    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
-    iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+    iex (irm https://community.chocolatey.org/install.ps1)
 
-    Write-Host "We are going to install some prerequisites now. This could take some time"
+    Write-Host "($(Get-Date -Format "dddd MM/dd/yyyy HH:mm:ss K")) We are going to install some prerequisites now. This could take some time"
     $null = choco install git pester visualstudio2022community visualstudio2022-workload-manageddesktop netfx-4.8-devpack -y
     Import-Module $env:ChocolateyInstall/helpers/chocolateyProfile.psm1
     Update-SessionEnvironment
-    Write-Host "Done installing software, now copying files"
+    Write-Host "($(Get-Date -Format "dddd MM/dd/yyyy HH:mm:ss K")) Done installing software, now copying files"
     # If you try to build on macOS, then run this, there are attributes on some files that cause the build to fail inside of the vagrant environment.
-    $null = robocopy c:/chocoRoot c:/code/choco /mir /copy:dt
+    $null = robocopy c:/chocoRoot c:/code/choco /e /copy:dt
+  SHELL
+  config.vm.provision "shell", name: "build", inline: <<-SHELL
+    $null = robocopy c:/chocoRoot c:/code/choco /e /copy:dt /purge
     Push-Location c:/code/choco
-    Write-Host "Files have been copied, beginning build process."
+    Write-Host "($(Get-Date -Format "dddd MM/dd/yyyy HH:mm:ss K")) Files have been copied, beginning build process."
     $CakeOutput = ./build.bat 2>&1
+
     if ($LastExitCode -ne 0) {
-      Set-Content c:/vagrant/buildOutput.log -Value $CakeOutput
-      Write-Host "The build has failed. Please see the buildOutput.log file for details"
-      exit $LastExitCode
+        Set-Content c:/vagrant/buildOutput.log -Value $CakeOutput
+        Write-Host "The build has failed. Please see the buildOutput.log file for details"
+        exit $LastExitCode
     }
 
-    Write-Host "Build complete. Executing tests."
+    Write-Host "($(Get-Date -Format "dddd MM/dd/yyyy HH:mm:ss K")) Build complete."
+  SHELL
+  config.vm.provision "shell", name: "test", inline: <<-SHELL
+    Write-Host "($(Get-Date -Format "dddd MM/dd/yyyy HH:mm:ss K")) Starting Test Execution"
+    Push-Location c:/code/choco
     # $env:TEST_KITCHEN = 1
     $env:VM_RUNNING = 1
-    ./Invoke-Tests.ps1
+    ./Invoke-Tests.ps1 -SkipPackaging

```

Comment:
Why are we skipping packaging here?

Reply:
@corbob we took the decision to remove this, as we felt that this wasn't necessary.  If if is, we can get this added back in.